docs/index.rst

```diff
@@ -0,0 +1,90 @@
+PreREISE
```

Comment:
Might be worth adding a link in this file to where the documentation source code is so it's easy for anybody reading it to know where to go to update the documentation. At least for me I've never worked with .rst files before 😅

Reply:
What kind of link you have in mind?

Reply:
https://github.com/Breakthrough-Energy/PreREISE/tree/develop/docs, but just a suggestion.

Reply:
Sounds good